CMakeLists.txt

```
@@ -294,9 +296,8 @@ foreach(kind ${kinds})
   add_library(${libTgt}_c OBJECT ${fms_c_src_files})
 
   target_include_directories(${libTgt}_c PRIVATE include
-                                                 grid_utils)
+                                                   grid_utils)
```

```suggestion
                                                 grid_utils)
```
consistent with line 435